compiler/codegen.js

```diff
@@ -4345,7 +4353,8 @@ const generateFunc = (scope, decl) => {
     returns: [ valtypeBinary, Valtype.i32 ],
     throws: false,
     name,
-    index: currentFuncIndex++
+    index: currentFuncIndex++,
+    hasRestArgument: false
```

Comment:
Don't need to explicitly set to `false` (as we make funcs in many places and having all the flags be explicit would be ehh)